assembl/graphql/post.py

```diff
@@ -327,6 +327,11 @@ def mutate(root, args, context, info):
         idea_id = int(Node.from_global_id(idea_id)[1])
         in_reply_to_idea = models.Idea.get(idea_id)
 
+        phase = in_reply_to_idea.get_associated_phase()
+        if not (phase.start < datetime.now() < phase.end):
+            error = _('It looks like you do not have the right to do this action. If you think it is an error, please reconnect to the platform and try again.')
```

Comment:
why not raise an error stating that the phase is closed and you can not participate ?